it/helpers/servicemocks/EISFinancialDataStub.scala

```diff
@@ -25,10 +24,10 @@ import play.api.libs.json.JsValue
 object EISFinancialDataStub {
 
   private def financialDataUrl(regime: TaxRegime, requestQueryParams: FinancialRequestQueryParameters): String =
-    if(requestQueryParams.hasQueryParameters) {
+    if(requestQueryParams.queryParams1811.nonEmpty) {
```

Comment:
The query params in the `else` case are still the old 1811 values. However, as your tests pass, this shows the `else` is not being used. This is because `queryParams1811` will always be nonEmpty as there are mandatory params that we always send for 1811. So I'd recommend simply removing the `if` statement and keeping the top value. This also means you can remove `requestQueryParams` argument from this function and the one below.